ITISFoundation/osparc-simcore
# StudyBrowser + bugs

### Waiting for library updates to be merged (Blocked)
## Changes
* Taking the image from the study thumbnail, whenever it is possible. If not, choosing one making use of the study's uuid
* Redesign of the dashboard + code refactoring
* Added new qxapp.ui.markdown.Markdown widget to display markdown. To be used maybe with the study description in the Dashboard... or anywhere else. **It is basically just a qx Label, with an extra markdown prop. If you use that one, it is setting the value property of the label with the compiled markdown. Also, it is taking markdown as its constructor's first argument**.
## Test it
Some good loving testing, please
* Try creating, opening (also from new button below the form), deleting (multiple) editing studies and templates.
* Check that everything works fine with permissions: disabling/displaying of buttons, allowed operations.
* Play with study selection. Only studies of the same section can be selected. Only user studies allow multi selection.
* Check the different tabs in the Dashboard
* Send me improving suggestions
## Screenshots
![Screenshot from 2019-08-06 15-23-37](https://user-images.githubusercontent.com/4764217/62543892-0cf07580-b85f-11e9-80b0-221ffeac3a81.png)
![Screenshot from 2019-08-06 15-28-16](https://user-images.githubusercontent.com/4764217/62543901-111c9300-b85f-11e9-99cc-292f67dbd8e0.png)
![Screenshot from 2019-08-06 15-24-11](https://user-images.githubusercontent.com/4764217/62543907-14b01a00-b85f-11e9-8296-d3f28fc260af.png)
![Screenshot from 2019-08-06 15-29-17](https://user-images.githubusercontent.com/4764217/62543913-17127400-b85f-11e9-8ce2-3e58afed14f7.png)
![Screenshot from 2019-08-06 15-24-45](https://user-images.githubusercontent.com/4764217/62543919-1aa5fb00-b85f-11e9-8b1c-3780b977a66f.png)
![Screenshot from 2019-08-06 15-29-27](https://user-images.githubusercontent.com/4764217/62543924-1d085500-b85f-11e9-9231-67a187ce6e3b.png)
